src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { start } from 'pretty-error';
+
+class TodoForm extends React.Component {
+    constructor(props){
+        super(props);
+        this.onSubmit = this.onSubmit.bind(this);
+    }
+
+    onSubmit(e){
+        e.preventDefault();
+        let newItem = this.refs.item.value;
```

Comment:
using refs isn't the desired solution here, you want to pass a function down from App.js on props and use that in this on `<form>` for `onSubmit`